maint/extractcvars.in

```diff
@@ -442,6 +466,20 @@ print OUTPUT_C <<EOT;
 }
 
 EOT
+
+foreach my $g (sort keys %enum_groups){
+    my $t = $enum_groups{$g};
+    print OUTPUT_C "int get_$g\_from_str(const char *s) {\n";
```

Comment:
Since this function can be called from device layer, should it be name-spaced?

Reply:
Agreed. How about `MPIR_$g\_from_str`?

Reply:
yes